src/containers/offer-page/specialization-block/SpecializationBlock.tsx

```diff
@@ -45,7 +45,25 @@ const SpecializationBlock = <T extends CreateOrUpdateOfferData>({
   const handleCheckboxesChange = (value: ProficiencyLevelEnum[]) => {
     handleNonInputValueChange('proficiencyLevel', value)
   }
-  const levelOptions = Object.values(ProficiencyLevelEnum)
+
+  const levelOptions = useMemo(() => Object.values(ProficiencyLevelEnum), [])
```

Comment:
do we really need useMemo here?

Reply:
This hook ensures that `levelOptions` won't be recreated during re-renders. In my opinion, this optimization prevents unnecessary re-renders of child components that depend on `levelOptions`.

Reply:
I think you can move both, ``levelOptions`` and ``levelLabels`` to constants and delete useMemo too, cause data is static

Reply:
Fix it

Reply:
Done, I've fixed it.